src/Isdc/Ui/Icons.elm

```diff
@@ -25,6 +26,10 @@ addCircleIconPath =
     "M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm5 11h-4v4h-2v-4H7v-2h4V7h2v4h4v2z"
 
 
+chevronRightIconPath =
```

Comment:
I think it would make more sense to put these next to the icon functions they relate to, or even in a ```let ... in ...```